src/lightning_app/utilities/port.py

```diff
@@ -14,7 +26,11 @@ def _find_lit_app_port(default_port: int) -> int:
     enable_multiple_works_in_default_container = bool(int(os.getenv("ENABLE_MULTIPLE_WORKS_IN_DEFAULT_CONTAINER", "0")))
 
     if not app_id or not project_id or not enable_multiple_works_in_default_container:
-        return default_port
+        # If the default port is not available, picks any other available one
+        if not is_port_in_use(default_port):
+            return default_port
+        else:
+            return get_available_port()
```

Comment:
@nohalon We could consider using the existing utility function `lightning_app.utilities.network.find_free_network_port`. This would be slightly faster compared to the current approach which opens a connection twice (once for checking port in use, once for getting a new one).

Reply:
perfect will use that! thanks @awaelchli